source/tools/dav.ts

```diff
@@ -233,7 +233,8 @@ export function parseSearch(result: DAVResult, searchArbiter: string, isDetailed
  * @returns The value in bytes, or another indicator
  */
 export function translateDiskSpace(value: string | number): DiskQuotaAvailable {
-    switch (value.toString()) {
+    value = String(value);
```

Comment:
Let's not re-assign function args - could you please make this a new constant?

Reply:
Or even better, just put the `String` call in the switch arg.

Reply:
makes sense, adjusted the code!